src/app/services/visit.service.ts

```diff
@@ -4,7 +4,7 @@ import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 
 import { Visit } from '../models/visit';
-import { environment } from '../../environments/environment.dev';
+import { environment } from '../../environments/environment';
```

Comment:
we have 3 files in environment folder:
![image](https://github.com/Johngtka/Biomagnethick-app/assets/11686968/c526d762-8e61-417e-bcfa-b1080094374e)

always use `environment.ts` in any import!